test/index.js

```diff
@@ -11,10 +11,11 @@ db.addMiddleware(addTimestamps);
 describe('Middleware timestamps', () => {
 	let dummyCollection;
 	before(() => {
```

Comment:
This includes an async call `db.create` and it should wait for this to finish before proceeding.
Check out http://mochajs.org/#working-with-promises for handling it.